### ISSUE TITLE
Line numbers display corrupted for some font sizes.

### ISSUE THREAD

**Issue by JonMcPhalen** [parallaxinc/Propeller-Tool#29]
When changing font size with Ctrl + Mouse Wheel, some sizes create problem for line numbers: 26, 32, 36, 42, and 48.
![font_scale_problem](https://user-images.githubusercontent.com/4042568/85045172-7873c680-b143-11ea-84d9-c9e57a66d92e.png)

**Comment by Wuerfel21** [comment]
That's actually a problem with the Parallax font. Just goes completely haywire on some font sizes. I think there exists a version somewhere that does not suffer from this?

**Comment by PropGit** [comment]
> That's actually a problem with the Parallax font. Just goes completely haywire on some font sizes. I think there exists a version somewhere that does not suffer from this?

Agreed - or a problem with the installation of the font.  Will explore when I can, may ask @JonMcPhalen to experiment with some steps.

For now, what version of Windows are you experiencing this on?

**Comment by JonMcPhalen** [comment]
I am using Windows 10 Pro

**Comment by cjmoorejr** [comment]
Here's a capture of the preview of the Parallax.ttf.  There's definitely something wrong with at least 18 & 24.  18 looks like it's only advancing to the next letter following certain characters like space.

I'm using Windows 10 Home 20H2.
 
![Capture](https://user-images.githubusercontent.com/36820917/100528262-62613c00-31a0-11eb-8a5b-719d7604b560.PNG)
